.travis.yml

```diff
@@ -127,9 +135,13 @@ matrix:
     allow_failures:
         # Do a PEP8 test with flake8
         # (do allow to fail unless your code completely compliant)
+        # - os: linux
+        #   stage: Initial tests
+        #   env: MAIN_CMD='flake8 dust_extinction --count --show-source --statistics $FLAKE8_OPT' SETUP_CMD=''
+
         - os: linux
-          stage: Initial tests
-          env: MAIN_CMD='flake8 dust_extinction --count --show-source --statistics $FLAKE8_OPT' SETUP_CMD=''
+          env: PYTHON_VERSION=3.6 ASTROPY_VERSION=lts NUMPY_VERSION=1.13 PYTEST_VERSION=<3.7
```

Comment:
Maybe need quotes?
```suggestion
          env: PYTHON_VERSION=3.6 ASTROPY_VERSION=lts NUMPY_VERSION=1.13 PYTEST_VERSION="<3.7"
```

Reply:
This was used as pytest was downgraded to 3.2.2 later in the travis build, so not sure the quotes are needed.